src/element/mutateElement.ts

```diff
@@ -53,8 +53,8 @@ export function newElementWith<TElement extends ExcalidrawElement>(
 ): TElement {
   return {
     ...element,
-    ...updates,
     version: element.version + 1,
     versionNonce: randomSeed(),
+    ...updates,
   };
```

Comment:
The way I'm allowing to retain versions in `history.generateEntry`.

Reply:
not sure about this change. if `version` changes the `versionNonce` should change with it. the main purpose of the `mutateElement()` function is to ensure that the version and nonce are bumped for every mutation so i don't think we should allow users to pass them in.

but for now let's just do this

Reply:
We can not touch either `versionNonce`/`version` during history pushing at all, how about that? I wasn't sure why we do so ITFP.